.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "image": "arohou/cistem_dev_env",   
+    "image": "aabhatti/xen_test",   
```

Comment:
This may be a valid commit, but it doesn't seem related to the topic of the PR. It might be nicer to restrict the PR only to those changes necessary to accomplish the goal (in this case, a new program to combine stacks)